### PR TITLE
Empty index xml in `MergedXml`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #415 
This PR fixes problem with empty index xml while appending rpms.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 please, take a look, I need this merged to continue the work

File: src/main/java/com/artipie/rpm/meta/MergedXmlPrimary.java [hunk 1]
```diff
@@ -132,6 +129,8 @@ public Result merge(final Map<Path, String> packages, final Digest dgst, final X
         boolean valid = true;
         final Collection<String> res = new ArrayList<>(locations.size());
         String checksum = "123";
+        reader.nextEvent();
+        reader.nextEvent();
```
Comment (g4s8) [inline src/main/java/com/artipie/rpm/meta/MergedXmlPrimary.java:133]: These lines look not clear: why do we need to skip two events here? I think we need to document it somehow.

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, updated method docs

**Pull request merged**